typescript/organisations.d.ts

```diff
@@ -226,3 +227,18 @@ export type ProductNotification = NewProductNotification & {
   createdAt: string
   updatedAt: string
 }
+
+export type NewAIDisclaimer = {
+  organisationId: string
+  type: 'FORMS_BUILDER'
+  user: UserProfile
+  data: {
+    aiFormsBuilderEnvironmentIds?: number[]
```

Comment:
```suggestion
    allowedFormsAppEnvironmentIds?: number[]
```

This will be a nested under `type: 'FORMS_BUILDER'`. I dont think we need to prefix this property with `aiFormsBuilder`.